website/docs/language/settings/backends/azurerm.mdx

```diff
@@ -295,11 +564,11 @@ When authenticating using the Storage Account's Access Key - the following field
 
 ***
 
-When authenticating using AzureAD Authentication - the following fields are also supported:
+When authenticating using Entra ID service principal, you have the option to use Entra ID authentication for the storage account (as opposed to Access Key or SAS Token) - the following fields are also supported:
```

Comment:
```suggestion
When authenticating using an Azure AD Service Principal, you have the option to use Azure Active Directory authentication for the storage account (rather than by an Access Key or SAS Token) - the following fields are also supported:
```